Makefile

```diff
@@ -1,48 +1,41 @@
 all: install
 
 ifeq ($(OS), Windows_NT)
-	Platform := "Windows"
+	Platform=Msys
+install:
+	mkdir $(USERPROFILE)\.cache 2> /dev/null
+	cp ani-cli $(WINDIR)/system32/ani-cli
+uninstall:
+	rm -rf $WINDIR/system32/ani-cli
 else
         Platform = $(shell uname -o)
 endif
 
-
 ifeq ($(Platform), GNU/Linux)
 	PREFIX := /usr/local
 install:
 	cp ani-cli $(DESTDIR)$(PREFIX)/bin/ani-cli
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
-else ifeq ($(Platform), Darwin)
+else
+ifeq ($(Platform), Darwin)
```

Comment:
Is else and ifeq on seperate lines somehow superior? 
I'd prefer them on the same line for better readability